### PR TITLE
feat(security): manage VO Group membership

### PR DESCRIPTION
**Pull request by fdlk**
#### Checklist
- [ ] Functionality works & meets specifications
- [ ] Code reviewed
- [ ] Code unit/integration/system tested
- [ ] User documentation updated
- [ ] Conventional commits (squash if needed)
- [ ] No warnings during install
- [ ] Updated javascript typing

### CODE REVIEW

File: packages/security/src/types/VOGroup.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export type VOGroup = {
```
Comment (connoratrug) [inline packages/security/src/types/VOGroup.ts:1]: what does 'VO' stand for ?

File: packages/security/src/components/GroupDetail.vue [hunk 2]
```diff
@@ -28,6 +28,33 @@
...
+        <h3 class="mt-2">{{'security-ui-vo-groups-header' | i18n}}</h3>
+      </div>
+    </div>
+
+    <div class="row groups-listing mt-1">
+      <div class="col">
+        <router-link
+          v-for="voGroup in voGroups"
+          :key="voGroup.groupName"
+          :to="{ name: 'memberDetail', params: { groupName: name, type: 'vo-group', memberName: voGroup.groupName } }"
```
Comment (fdlk) [inline packages/security/src/components/GroupDetail.vue:46]: let's switch to ID here

File: packages/security/src/types/VOGroup.ts (hunk 1 above)
Comment (fdlk) [reply packages/security/src/types/VOGroup.ts:1]: yes, we definitely need more docs :)

Comment (fdlk) [reply packages/security/src/types/VOGroup.ts:1]: It stands for Virtual Organisation, a remotely managed group of people

File: packages/security/src/store/actions.ts [hunk 3]
```diff
@@ -103,9 +138,22 @@ const actions = {
     })
   },
 
-  'addMember' ({ commit }: { commit: Function }, { groupName, addMemberCommand }: {groupName: string; addMemberCommand: AddMemberCommand}) {
-    const url = GROUP_ENDPOINT + '/' + encodeURIComponent(groupName) + '/member'
-    const payload = { body: JSON.stringify(addMemberCommand) }
+  'addMember' ({ commit, state }: { commit: Function; state: { voGroups: VOGroup[] | null} }, { groupName, type, name, roleName }: {groupName: string; type: 'member'|'vo-group'; name: string; roleName: string}) {
```
Comment (jelmerveen) [inline packages/security/src/store/actions.ts:141]: add VO typing, to make it more readable

**Review by jelmerveen — APPROVED** [review]

**Pull request merged**